### DISCUSSION TITLE
Tuya circuit breaker(TS0601, _TZE200_fsb6zw01) returns wrong energy value

### DISCUSSION THREAD

**Discussion by nghetienhiep** [Koenkk/zigbee2mqtt#14854] [Devices]
I have Tuya's circuit breaker. All parameters like (voltage, power, current, ...) work correctly, but it seems that the energy value is returning x10 with the correct value. What do I have to do now? I need some help!

![image](https://user-images.githubusercontent.com/13849419/200346616-d9e8d460-1848-4683-abe7-a296356c5b69.png)
![image](https://user-images.githubusercontent.com/13849419/200346115-80149281-12ed-4a9c-8597-ac4251177f5e.png)
![image](https://user-images.githubusercontent.com/13849419/200346236-11bd9307-9650-480b-abe6-4ab54754df57.png)

**Comment by m2LiJu** [comment]
Hi I have exactly same device and same problem. I had to create external converter for this to get 10 times lower energy. 
The problem is that external loader stops working few months ago with error in logs:
[Fail to start - Cannot read properties of undefined (reading ‘options’)]

Any chance to fix this somehow?
Regards
